lib/hotbar.js

```diff
@@ -1,6 +1,31 @@
 const { LitElement, html, css } = require('lit-element')
 const invsprite = require('./invsprite.json')
 
+const itemsSprite = document.createElement('img')
+itemsSprite.src = 'invsprite.png'
+const itemIcons = new Map()
+
+const createItemIcon = async (item) => {
```

Comment:
Don't use canvas to draw the item icons, its better to let css do it.

Reply:
It's better to be able to scale. And it's only draws on time so it's ok.

Reply:
It was already able to scale, please revert to use css.

Reply:
It wasn't.

Reply:
There was multiple way to do it. The easiest one was using css transforms to set the scale globally (adding a single line to scale the whole ui). Creating multiple canvas even if you cache them, takes more memory than just letting css do its thing and make the code harder to read.